Generator/Generator/BuiltinGen.swift

```diff
@@ -259,8 +269,9 @@ func generateMethodCall (_ p: Printer,
             p ("\(typeName).\(methodToCall) (nil, \(ptrArgs), \(ptrResult)\(numberOfArgs))")
     } else {
         if isStructMap [typeName] ?? false {
-            p ("withUnsafePointer (to: self) { ptr in ")
-            p ("    \(typeName).\(methodToCall) (UnsafeMutableRawPointer (mutating: ptr), \(ptrArgs), \(ptrResult)\(numberOfArgs))")
+            p ("var mutSelfCopy = self")
+            p ("withUnsafeMutablePointer (to: &mutSelfCopy) { ptr in ")
+            p ("    \(typeName).\(methodToCall) (ptr, \(ptrArgs), \(ptrResult)\(numberOfArgs))")
             p ("}")
```

Comment:
Question, what was the rationale for this change?  

We have two scenarios, one, Godot will never mutate on these calls, so in that case, this seems unnecessary.   But if there is ever a mutation of the code, then this would discard the mutation.   So I want to understand this, as it seems like this could be a semantic change that could become tricky to diagnose.

Reply:
The rationale behind this change is that just like const-cast (which `.init(mutating:)` is) the a mutation done by mistake by Godot won't be _just_ discarded, it's in fact a UB.
May be it's indeed super-defensive programming.